ivadomed/scripts/training_curve.py

```diff
@@ -145,33 +144,33 @@ def run_plot_training_curves(input_folder, output_folder, multiple_training=Fals
         n_cols, n_rows = 1, 1
 
     for i_subplot, input_folder in enumerate(group_list):
-        input_folder = os.path.expanduser(input_folder)
+        input_folder = Path(input_folder).expanduser()
         # Find training folders:
         if multiple_training:
-            prefix = str(input_folder.split('/')[-1])
-            input_folder = '/'.join(input_folder.split('/')[:-1])
-            input_folder_list = [os.path.join(input_folder, f) for f in os.listdir(input_folder) if
-                                 f.startswith(prefix)]
+            prefix = input_folder.name
+            input_folder = input_folder.parent
```

Comment:
VERY nice! 👍 
Fix like this is how we can ensure MUCH better cross platform compatibility.